man/org.freedesktop.resolve1.xml

```diff
@@ -0,0 +1,669 @@
+      <constant>AF_INET6</constant>, <constant>AF_UNSPEC</constant>), if this is enabled (Use the
+      <constant>NO_ADDRESS</constant> flag to turn address resolution off, see below). The
+      <varname>flags</varname> parameter takes a couple of flags that may be used to alter the resolver
+      operation.</para>
+
+      <para>On return, <function>ResolveService()</function> returns an array of SRV record structures. Each
+      item consists of the priority, weight and port fields and the hostname to contact, as encoded in the SRV
+      record. Immediately following is an array with the addresses of this hostname, with each item consisting
+      of the interface index, the address family and the address data in a byte array. This address array is
+      followed with the canonicalized hostname. After this array of SRV record structures an array of byte
+      arrays follows, that encodes the TXT RR strings, in case DNS-SD look-ups are enabled. The next parameters
+      are the canonical service name, type and domain. This may or may not be identical to the parameters
+      passed in. Finally, a <varname>flags</varname> field is returned that contains information about the
+      resolver operation performed.</para>
+
+      <para>The <function>ResetStatistics()</function> method resets to zero the various statistics counters
+      <filename>systmed-resolved</filename> maintains. (For details, see the statistics properties below.)</para>
+
+      <para>The <function>GetLink()</function> method takes a network interface index and returns the object
+      path to the <interfacename>org.freedesktop.resolve1.Link</interfacename> object corresponding to it.
+      </para>
+
+      <para>The <function>SetLinkDNS()</function> method sets the DNS servers to use on a specific
+      interface. This call (and the following ones) may be used by network management software to configure
+      per-interface DNS settings. It takes a network interface index as well as an array of DNS server IP
+      address records. Each array item consists of an address family (either <constant>AF_INET</constant> or
+      <constant>AF_INET6</constant>), followed by a 4-byte or 16-byte array with the raw address data. This
+      call is a one-call shortcut for retrieving the Link object for a network interface using
+      <function>GetLink()</function> (see above) and then invoking the <function>SetDNS()</function> call
+      (see below) on it.
+      </para>
+
+      <para>Network management software integrating with <filename>systmed-resolved</filename> is recommended
+      to invoke this method (and the five below) after the interface appeared in the kernel (and thus after a
+      network interface index has been assigned), but before the network interfaces is activated (set
+      <constant>IFF_UP</constant> on) so that all settings take effect during the full time the network
```

Comment:
```suggestion
      network interface index has been assigned), but before the network interfaces is activated (
      <constant>IFF_UP</constant> set on) so that all settings take effect during the full time the network
```